lua/projectlaunch/jobs.lua

```diff
@@ -20,8 +24,8 @@ function Job:new(command, opts)
 		-- duplicating the name lets commands be edited and
 		-- jobs won't get the updated command until it's rerun,
 		-- so it won't look like the newly edited command had been run automatically
-		name = command.name,
-		cmd = command.cmd,
+		name = runtimeVarManager.generateJobName(command),
+		cmd = finalCommand,
```

Comment:
The possible future issues I was referring to in [my other comment](https://github.com/sheodox/projectlaunch.nvim/pull/6/files#r1319170566) was talking about a situation like if I'm refactoring this code sometime in the future and forget about this conversation, then if I were to remove creating `finalCommand` on line 11, then moved the interpolate call to this `cmd = ` line, then just reused `j.cmd` instead of finalCommand on line 37.

That would seem like a harmless way to refactor this as it removes creating a variable and the rest looks equivalent. But then you'd be not showing the right name, because `vars` in the runtime variable manager module is some module local variable that stores only the last `interpolate` call's variables, so reordering those calls will make it show variables for the **previous** command, then people are running the wrong command without realizing it.